CMakeLists.txt

```diff
@@ -689,6 +684,11 @@ target_link_libraries(ktx PRIVATE ${ASTC_LIB_NAME})
 # Its a good optimisation anyways and most compilers now support it
 set_property(TARGET ktx PROPERTY INTERPROCEDURAL_OPTIMIZATION_RELEASE True)
 
+# Use a static runtime on MSVC builds (ignored on non-MSVC compilers)
+set_property(TARGET ktx
+    PROPERTY
+        MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
```

Comment:
What about the Release config?

Make sure multi-threaded isn't set when building a static libktx.

Reply:
> What about the Release config?

Reply https://github.com/KhronosGroup/KTX-Software/pull/433#discussion_r658677332

> Make sure multi-threaded isn't set when building a static libktx.

Why not? This is the static version of MSVC runtime library.

Reply:
This has now disappeared because we have added a `CLI=OFF` option to astcencoder.